### ISSUE TITLE
ppc64le: Issue with gcc-12 and gcc-13

### ISSUE THREAD

**Issue by RajalakshmiSR** [conda-forge/ctng-compilers-feedstock#131]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue
```
gcc -o t t.c -mcpu=native
cc1: error: unrecognized argument in option '-mcpu=native'
cc1: note: valid arguments to '-mcpu=' are: 401 403 405 405fp 440 440fp 464 464fp 476 476fp 505 601 602 603 603e 604 604e 620 630 740 7400 7450 750 801 821 823 8540 8548 860 970 G3 G4 G5 a2 cell e300c2 e300c3 e500mc e500mc64 e5500 e6500 ec603e power10 power3 power4 power5 power5+ power6 power6x power7 power8 power9 powerpc powerpc64 powerpc64le rs64 titan
```

### Installed packages

```shell
# packages in environment at /home/raji/anaconda3/envs/gccissue1:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       2_gnu    conda-forge
_sysroot_linux-ppc64le_curr_repodata_hack 4                   h43410cf_14    conda-forge
binutils_impl_linux-ppc64le 2.40                 hba8546e_0    conda-forge
bzip2                     1.0.8                ha17a0cc_5    conda-forge
ca-certificates           2024.2.2             h0f6029e_0    conda-forge
gcc                       13.2.0               hb8c6617_3    conda-forge
gcc_impl_linux-ppc64le    13.2.0               h7908001_5    conda-forge
kernel-headers_linux-ppc64le 3.10.0              h23d7e6c_14    conda-forge
ld_impl_linux-ppc64le     2.40                 hfa29eb9_0    conda-forge
libffi                    3.4.2                h4e0d66e_5    conda-forge
libgcc-devel_linux-ppc64le 13.2.0             hf1adbf4_105    conda-forge
libgcc-ng                 13.2.0               h31ea8bd_5    conda-forge
libgomp                   13.2.0               h31ea8bd_5    conda-forge
libnsl                    2.0.1                ha17a0cc_0    conda-forge
libsanitizer              13.2.0               h0849fd4_5    conda-forge
libsqlite                 3.45.3               hd4bbf49_0    conda-forge
libstdcxx-ng              13.2.0               h0849fd4_5    conda-forge
libuuid                   2.38.1               h4194056_0    conda-forge
libxcrypt                 4.4.36               ha17a0cc_1    conda-forge
libzlib                   1.2.13               ha17a0cc_5    conda-forge
ncurses                   6.4.20240210         h8800142_0    conda-forge
openssl                   3.2.1                ha17a0cc_1    conda-forge
pip                       24.0               pyhd8ed1ab_0    conda-forge
python                    3.10.14         h4005451_0_cpython    conda-forge
readline                  8.2                  h0b9b154_1    conda-forge
setuptools                69.5.1             pyhd8ed1ab_0    conda-forge
sqlite                    3.45.3               h63c7444_0    conda-forge
sysroot_linux-ppc64le     2.17                h23d7e6c_14    conda-forge
tk                        8.6.13               hd4bbf49_0    conda-forge
tzdata                    2024a                h0c530f3_0    conda-forge
wheel                     0.43.0             pyhd8ed1ab_1    conda-forge
xz                        5.2.6                hb283c62_0    conda-forge
zlib                      1.2.13               ha17a0cc_5    conda-forge
```

### Environment info

```shell

     active environment : gccissue1
    active env location : /home/raji/anaconda3/envs/gccissue1
            shell level : 2
       user config file : /home/raji/.condarc
 populated config files : /home/raji/.condarc
          conda version : 22.9.0
    conda-build version : 3.22.0
         python version : 3.9.13.final.0
       virtual packages : __linux=4.18.0=0
                          __glibc=2.28=0
                          __unix=0=0
                          __archspec=1=ppc64le
       base environment : /home/raji/anaconda3  (writable)
      conda av data dir : /home/raji/anaconda3/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/linux-ppc64le
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/linux-ppc64le
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-ppc64le
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/raji/anaconda3/pkgs
                          /home/raji/.conda/pkgs
       envs directories : /home/raji/anaconda3/envs
                          /home/raji/.conda/envs
               platform : linux-ppc64le
             user-agent : conda/22.9.0 requests/2.28.1 CPython/3.9.13 Linux/4.18.0-513.24.1.el8_9.ppc64le rhel/8.9 glibc/2.28
                UID:GID : 1011:1011
             netrc file : None
           offline mode : False


```

**Comment by RajalakshmiSR** [comment]
@isuruf When will this fix be available as part of conda-forge build?

**Comment by h-vetinari** [comment]
> When will this fix be available as part of conda-forge build?

After you update `gcc`, it should work already. The fixed builds got released a few days ago with #132.
